package.json

```diff
@@ -39,6 +39,8 @@
     "docs:dist": "npm run clean:dist && NODE_ENV=production webpack --progress --display=minimal --bail && npm run sassdoc",
     "start": "npm run build && npm run storybook",
     "native": "./subshell ./native npm run storybook",
+    "generate-changelogs": "node scripts/release-process/generate-changelogs.js",
+    "upcoming-changes": "node scripts/release-process/upcoming-changes.js",
```

Comment:
Not part of the card probably, but we might want to standardise how we handle scripts, we have a mixture of `node {script}.js` and executable files